remote_monitoring_demo/json_files/update_results.json

```diff
@@ -1,96 +1,102 @@
-[{
- "experiment_name": "quarkus-resteasy-kruize-min-http-response-time-db",
- "start_timestamp": "2022-01-23T18:25:43.511Z",
- "end_timestamp": "2022-01-23T18:25:43.511Z",
- "deployments": [{
-   "deployment_name": "tfb-qrh-sample",
-   "namespace": "default",
-   "pod_metrics": [],
-   "containers": [{
-     "image_name": "kruize/tfb-qrh:1.13.2.F_et17",
-     "container_name": "tfb-server",
-     "container_metrics": {
-       "cpuRequest" : {
-         "results": {
-           "general_info": {
-             "sum": 4.4,
-             "avg": 1.1,
-             "units": "cores"
-           }
-         }
-       },
-       "cpuLimit": {
-         "results": {
-           "general_info": {
-             "sum": 2.0,
-             "avg": 0.5,
-             "units": "cores"
-           }
-         }
-       },
-       "cpuUsage": {
-         "results": {
-           "general_info": {
-             "sum": 2.14,
-             "min": 0.14,
-             "max": 0.84,
-             "avg": 0.12,
-             "units": "cores"
-           }
-         }
-       },
-       "cpuThrottle": {
-         "results": {
-           "general_info": {
-             "sum": 0.19,
-             "max": 0.09,
-             "avg": 0.045,
-             "units": "cores"
-           }
-         }
-       },
-       "memoryRequest": {
-         "results": {
-           "general_info": {
-             "sum": 250.85,
-             "avg": 50.21,
-             "units": "MiB"
-           }
-         }
-       },
-       "memoryLimit": {
-         "results": {
-           "general_info": {
-             "sum": 500,
-             "avg": 100,
-             "units": "MiB"
-           }
-         }
-       },
-       "memoryUsage": {
-         "results": {
-           "general_info": {
-             "min": 50.6,
-             "max": 198.50,
-             "sum": 198.50,
-             "avg": 40.1,
-             "units": "MiB"
-           }
-         }
-       },
-       "memoryRSS": {
-         "results": {
-           "general_info": {
-             "min": 50.6,
-             "max": 123.6,
-             "sum": 123.6,
-             "avg": 31.91,
-             "units": "MiB"
-           }
-         }
-       }
-     }
-   }]
- }]
-}]
+[
+  {
+    "experiment_name": "quarkus-resteasy-kruize-min-http-response-time-db",
+    "start_timestamp": "2022-01-23T18:25:43.511Z",
```

Comment:
@msvinaykumar - I see you have changed the format for timestamp. The format of timestamp that the json had earlier was that present in tfb_data.csv  The timestamp (2022-11-25 21:00:00 +0000 UTC) given by nise tool or in customer data shared is also different. Shouldn't we be doing the conversion internally in the code?

Reply:
Usually date, time, and time zone are represented and processed as per http://en.wikipedia.org/wiki/ISO_8601.
I chose the ISO-8601 standard because it is more generic and will be used by applications other than ROS. Let me know if this helps.